Example/Podfile

```diff
@@ -3,6 +3,7 @@ platform :ios, '14.1'
 use_frameworks!
 
 target 'ReadDiceKey_Example' do
+  pod 'OpenCVXF', :path => '../../opencv-xf'
```

Comment:
@bakhtiyork 
Relative paths go outside the repository.  Does this assume a certain structure to where the developer will install their different repositories? Does this need to be fixed?